.github/readme.md

```diff
@@ -23,47 +23,95 @@ If you want to contribute as a developer, please contact us in the [Discord](htt
 
 ## Building
 
-First, install the dependencies for Debian-like distributions.
-   
-``sudo apt install build-essential cmake libunwind-dev libglfw3-dev libvulkan-dev vulkan-validationlayers-dev spirv-tools glslang-tools libspirv-cross-c-shared-dev libxbyak-dev``
+
+### The dependencies for Debian-like distributions.
+```   
+sudo apt install build-essential cmake libunwind-dev libglfw3-dev libvulkan-dev vulkan-validationlayers-dev spirv-tools glslang-tools libspirv-cross-c-shared-dev
+```
+
+### The dependencies for Fedora distributions:
+
+```
+sudo dnf install cmake libunwind-devel glfw-devel vulkan-devel vulkan-validation-layers-devel spirv-tools glslang-devel gcc-c++ gcc spirv-tools-devel xbyak-devel
+```
+
+### The dependencies for Arch distributions:
+```
+sudo pacman -S libunwind glfw-x11 vulkan-devel glslang
+```
+> Side note you will need to pull ``spirv-cross xbyak`` from the AUR for now so do the following
+
+```
+sudo pacman -S --needed git base-devel && git clone https://aur.archlinux.org/yay.git && cd yay && makepkg -si
+```
+```
+yay -S spirv-cross xbyak
+```
+
+
+## Getting spriv-cross on Fedora and Arch Linux
+
+```
+git clone https://github.com/KhronosGroup/SPIRV-Cross && cd SPIRV-Cross && mkdir build && cd build && cmake .. && cmake --build . && sudo make install
+```
+> **Warning** <br/>
+> Fedora will compile to a point and then error out
+> Arch will have to use xbyak from the aur for now
 
 ## Cloning the Repo
 
-``git clone https://github.com/RPCSX/rpcsx && cd rpcsx``
+```
+git clone --recursive https://github.com/RPCSX/rpcsx && cd rpcsx
+```
+
+> if you get a cmake error run
```

Comment:
Does it really need the > here? I viewed it on your branch abd it looks weird imo.

Reply:
nope not anymore the issue was solved